src/main/java/io/github/moulberry/notenoughupdates/util/ProfileApiSyncer.java

```diff
@@ -89,9 +90,15 @@ private void forceResync() {
 		if (Minecraft.getMinecraft().thePlayer == null) return;
 
 		String uuid = Minecraft.getMinecraft().thePlayer.getUniqueID().toString().replace("-", "");
-		NotEnoughUpdates.profileViewer.getOrLoadSkyblockProfiles(uuid, (profile) -> {
-			for (Consumer<SkyblockProfiles> c : finishSyncCallbacks.values()) c.accept(profile);
-			finishSyncCallbacks.clear();
-		});
+		NotEnoughUpdates.INSTANCE.manager.apiUtils
+			.newHypixelApiRequest("/skyblock/profiles")
```

Comment:
Shouldn't this either use ursa or an anonymous request (not sure which one it should be here)?

Reply:
Neither is applicable. Ursa minor should be used for the pv (not for passive background polling) and you need an api key to make this request.